src/MovieDisplay/MovieDisplay.js

```diff
@@ -22,47 +24,55 @@ class MovieDisplay extends Component {
     fetchData(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.params.id}`)
       .then((data) => {
         this.setState({selectedMovie: data.movie, isLoading: false})
+      }).then(() => {
+        if (this.props.checkFavorites(this.state.selectedMovie.id)) {
+          this.setState({isFavorite: true})
+        }
       })
       .catch((err) => this.setState({err, isLoading:false}));
   }
 
-  //these needed to be chained.
   addToFavorites = () => {
-    postData(this.state.selectedMovie).then(() => this.props.getFavorites())
+    postData(this.state.selectedMovie).then(() => this.props.getFavorites());
   }
 
   removeFromFavorites = () => {
-    deleteData(this.state.selectedMovie.id).then(() => this.props.getFavorites())
-    // deleteData('delete').then(() => this.props.getFavorites())
+    deleteData(this.state.selectedMovie.id).then(() => this.props.getFavorites());
   }
 
   render() {
     const movie = (
       <section className='MovieDisplay' >
+      <div className="buttonContainer">
```

Comment:
Great job setting up this conditional rendering and passing props from our parent to make our post request. Looks very clean and easy to read. Thank you!